wikimedia/wikipedia-ios
# [YiR] Metrics Part 2

**Phabricator:** https://phabricator.wikimedia.org/T376302

### Notes
This PR adds another set of metrics for year in review. Any additional instrumentation after this point will be done on a task-by-task basis (for example: https://phabricator.wikimedia.org/T376350 and https://phabricator.wikimedia.org/T379825)

### Test Steps
1. Temporarily update the WMFLogging.h [value](https://github.com/wikimedia/wikipedia-ios/compare/main...yir-metrics-2#diff-1995380b046bbbb2010885e0d1b6b817046bfd60db4f4f6b5a70d6d93de4d421R6) to DDLogLevelAll, so that we will see event payloads in the Xcode console.
2. Observe the share button taps, closing donate slide button taps, and survey event payloads, and lightly confirm they look correct according to the [deck](https://docs.google.com/presentation/d/1uAiqSlb_vpJsW2KJSM2X38JCDflRTOI2b57ygtad45I/edit#slide=id.g249f09432e1_0_0). (Note: We can lean on QA and beta tester validation for more in-depth testing).
